packages/server/modules/automate/rest/logStream.ts

```diff
@@ -21,27 +19,30 @@ import { Application } from 'express'
 
 export default (app: Application) => {
   app.get(
-    '/api/automate/automations/:automationId/runs/:runId/logs',
+    '/api/v1/projects/:projectId/automations/:automationId/runs/:runId/logs',
```

Comment:
is it fine breaking this API?

Reply:
yes, its only the frontend, that uses this api

Reply:
yes has been adjusted to reflect this 👍